internal/server/ranking/ranking.go

```diff
@@ -105,53 +108,59 @@ type CohortByRank []*pb.SourceSeries
 // score, otherwise can also match to wildcard options (indicated by *).
 //
 // If no entry is found, a BaseRank is assigned to the source series.
+func GetScoreRk(importName string, rk RankKey) int {
+    importNameStatsRanking, ok := StatsRanking[importName]
+    if ! ok {
+        return BaseRank
+    }
+    scoreOfMostExactMatch := BaseRank
+    mostMatches := -1
+    for k, score := range importNameStatsRanking {
+        matches := 0
+        isMatch := true
+        if k.MeasurementMethod != "*" {
```

Comment:
if k.MeasurementMethod == "*", then should matches++?

Reply:
If you will handle this, and also make sure exact match is better than * match, then can increment * match by 0.9, and exact match by 1 (changing matches from int to float).

Reply:
No, we use "*" as a placeholder and do not want to increment matches for "*" fields. We use "*" as a placeholder and do not count it towards a match specifically because we want more exact matches to be scored higher.

Incrementing "*" matches would not change behavior because in case of at least 1 non-placeholder field, that gets matched and becomes the score. In case all RankKey fields are "*"; the first field will match because line 117: `mostMatches := -1`.

Reply:
moving forward with changing `RankKey` fields to be string pointers and checking for nil-ness.